SnookR/sublist/urls.py

```diff
@@ -2,5 +2,6 @@
 from . import views
 
 urlpatterns = [
+    url(r'register_sub$', views.RegisterSubView.as_view(), {'next_page': 'home'}, name='register_sub'),
```

Comment:
The Django convention is to have a trailing slash on all URLs.  Why this is, I do not know (and I'm not experienced enought to guess), but I would assume it's best to conform.